great_expectations/data_context/store/ge_cloud_store_backend.py

```diff
@@ -380,6 +393,12 @@ def config(self) -> dict:
 
     def build_key(self, id_: Optional[str] = None, **kwargs) -> GeCloudIdentifier:
         """Get the store backend specific implementation of the key, ignore irrelevant kwargs."""
+        if "name" in kwargs:
+            return GeCloudIdentifier(
+                resource_type=self.ge_cloud_resource_type,
+                ge_cloud_id=id_,
+                resource_name=kwargs["name"],
+            )
```

Comment:
Ah this is tough - I don't love kwargs but I see the need for them here. Any way we can avoid and use an actual arg?

Reply:
I was about to ask the same thing. Does this need `kwargs`? We aren't using them.
```python
def build_key(self, id_: Optional[str] = None, name: Optional[str] = None) -> GeCloudIdentifier:
```

Reply:
Good call 👍